pkgdown-netlify-preview/pkgdown-netlify-preview.yaml

```
@@ -53,13 +53,20 @@ jobs:
           clean: false
           branch: gh-pages
           folder: docs
-
+      - id: deploy-dir 
+        name: Determine dev status
+        run: |
+          if [[ $(grep -c -E 'sion. ([0-9]*\.){3}' ${{ github.workspace }}/DESCRIPTION) == 1 ]]; then
+            echo 'dir=./docs/dev' >> $GITHUB_OUTPUT
+          else
+            echo 'dir=./docs' >> $GITHUB_OUTPUT
+          fi
       - name: Deploy PR preview to Netlify
         if: contains(env.isPush, 'false')
         id: netlify-deploy
         uses: nwtgck/actions-netlify@v2
```

```suggestion
        uses: nwtgck/actions-netlify@v3
```
Might as well update this action too which is on v3 release? https://github.com/nwtgck/actions-netlify/releases/tag/v3.0.0